pocketbooksync.koplugin/main.lua

```diff
@@ -54,52 +54,46 @@ function PocketbookSync:prepareSync()
         return nil
     end
 
-    local cacheKey = self.ui.document.file
-
-    if not preparedData[cacheKey] then
```

Comment:
There's still remnants of `preparedData` elsewhere in the file. Perhaps it would've been wiser to simply revert f5f7d3879a45d781a94e09fd327b24e66da10db2 first, and then implement the other changes in a separate commit?

Reply:
Removed the remnants now. I like to have scheduled sync to be removed. Please see my other comment for motivation.